frontend/app/views/spree/shared/_locale_and_currency.html.erb

```diff
@@ -1,6 +1,7 @@
 <script>
   var SPREE_LOCALE = '<%= I18n.locale %>'
   var SPREE_CURRENCY = '<%= current_currency %>'
+  var SPREE_CURRENCY_SYMBOL = '<%= currency_symbol(current_currency) %>'
```

Comment:
@damianlegawiec Please let me know if that's a fine approach. The `currency_symbol` method sits in the `Spree::CurrencyHelper` module.